src/lib/components/FormModel/children/Form/UI/DateTimePicker/date-time-picker.component.js

```diff
@@ -14,16 +14,16 @@ export const DateTimePicker = props => {
   const { id, data, updateData } = props;
 
   const [selectedDate, setDate] = useState(null);
-  const [invalidate, setInvalid] = useState(null);
 
   const { theme } = useContext(ThemeContext);
 
   useEffect(() => {
     setDate(parseInitialValue(data[UI.VALUE], data[RDF.TYPE]));
   }, [data[UI.VALUE]]);
 
-  const minValue = data[UI.MINVALUE];
-  const maxValue = data[UI.MAXVALUE];
+  // Fetch relevant values from the data prop, which represents the properties in the form model
+  const minValue = data[UI.MIN_VALUE];
+  const maxValue = data[UI.MAX_VALUE];
```

Comment:
I'm not sure whether these values come from, but shouldn't we see the variable name change occur where they're defined as well?

Reply:
No, this is the root cause of the bug (the name has _ in them, but it was not entered correctly here).

Reply:
To add a little more detail, `UI` is located inside of `/constants/index.js` and already had these defined. Previously, it was set to `UI.MINVALUE` instead of `UI.MIN_VALUE` and this obviously caused some issues as that was undefined.